packages/core/bootstrap/README.md

```diff
@@ -160,11 +160,12 @@ Being:
 
 \*To use this feature the `CACHE_ENABLED` environment variable must also be enabled.
 
-| Required? |             Name              |                                                                          Description                                                                          | Options |    Defaults to     |
-| :-------: | :---------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :----------------: |
-|           | `EXPERIMENTAL_WARMUP_ENABLED` |                                                            Enable the cache warmer functionality.                                                             |         |      `false`       |
-|           | `WARMUP_UNHEALTHY_THRESHOLD`  |                      The number of times a warmup execution can fail before we drop a warmup subscription for a particular cache key.to.                      |         |        `3`         |
-|           |   `WARMUP_SUBSCRIPTION_TTL`   | The maximum duration between requests for a cache key to an external adapter before the cache warmer will unsubscribe from warming up a particular cache key. |         | `3600000` (1 hour) |
+| Required? |             Name              |                                                                                      Description                                                                                      | Options |      Defaults to      |
+| :-------: | :---------------------------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: | :-------------------: |
+|           | `EXPERIMENTAL_WARMUP_ENABLED` |                                                                        Enable the cache warmer functionality.                                                                         |         |        `false`        |
+|           | `WARMUP_UNHEALTHY_THRESHOLD`  |                                  The number of times a warmup execution can fail before we drop a warmup subscription for a particular cache key.to.                                  |         |          `3`          |
```

Comment:
Added the ability to disable `WARMUP_UNHEALTHY_THRESHOLD`.

This is useful when we know that we do want to continue warming this data as long as requests are coming in (controlled by `WARMUP_SUBSCRIPTION_TTL`), and we should continue trying to warm the cache even if the data provider goes down for some amount of time.